include/odp/api/spec/ipsec.h

```diff
@@ -337,6 +337,28 @@ typedef struct odp_ipsec_capability_t {
 	 */
 	uint32_t max_cls_cos;
 
+	/**
+	 * Scheduled queue support
+	 *
+	 * 0: Scheduled queues are not supported as IPsec SA destination queues
+	 *    or IPsec default queue
+	 * 1: Scheduled queues are supported as IPsec SA destination queues or
```

Comment:
The 'or' should be 'and'. If this capability field is true, then scheduled queues are supported both as destination queues and as the default queue. And the same comment for the plain queue capability below.

Reply:
In v4, updated the text as follows:

```
+        * 0: Scheduled queues are not supported either as IPsec SA destination
+        *    queues or as IPsec default queue
+        * 1: Scheduled queues are supported as both IPsec SA destination queues
+        *    and IPsec default queue
```

Let me know if this looks OK.